python/cudf/cudf/utils/applyutils.py

```diff
@@ -104,10 +127,15 @@ def run(self, df, **launch_params):
         bound = self.sig.bind(**args)
         # Launch kernel
         self.launch_kernel(df, bound.args, **launch_params)
+        # Prepare pessimistic nullmask
+        out_mask = make_aggregate_nullmask(df)
```

Comment:
I think this should be `op="or"`, since you want a bitwise or so that if any value is null it outputs a null, right?

Reply:
Or whatever the op name is for a bitwise or (as opposed to a logical or)

Reply:
The flipped but represents a valid value, so bitwise-and represents where all the aligned values are valid. Do I have this backwards? The tests seem to be passing..

Reply:
You're absolutely correct, thanks for clarifying for me! Resolving this